src/mjolnir/elevationbuilder.cc

```diff
@@ -63,102 +204,7 @@ void add_elevation(const boost::property_tree::ptree& pt,
     tilequeue.pop_front();
     lock.unlock();
 
-    // Get the tile. Serialize the entire tile?
-    GraphTileBuilder tilebuilder(graphreader.tile_dir(), tile_id, true);
```

Comment:
this chunk was just moved out into the method above right? were there any actual changes to the code block?

Reply:
no changes at this particular part, i just needed to reuse this logic in another method ["load_tile_elevations"](https://github.com/valhalla/valhalla/blob/4a991391bfc532afeecbb5e84e802aca975beb0a/valhalla/mjolnir/elevationbuilder.h#L19) and so decided to move it to a separate method so it will help me to avoid code repetition by reusing available one.